src/components/shop-app.js

```diff
@@ -358,88 +368,71 @@ class ShopApp extends Element {
     // this.addEventListener('announce', (e)=>this._onAnnounce(e));
     // this.addEventListener('dom-change', (e)=>this._domChange(e));
     // this.addEventListener('show-invalid-url-warning', (e)=>this._onFallbackSelectionTriggered(e));
+
+    const mq = window.matchMedia('(max-width: 767px)');
+    mq.addListener(_ => this._smallScreen = mq.matches);
+    this._smallScreen = mq.matches;
+  }
+
+  _categoryNameChanged(categoryName, oldCategoryName) {
+    if (categoryName !== oldCategoryName) {
```

Comment:
I don't think we need the check since `_categoryNameChanged` will only be called if `categoryName` has changed.